news/2020-11-19-better-protecting-players-from-abuse.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Better protecting players from abuse"
+date: 2020-11-19 09:00:00 +0000
+---
+
+Hi everyone, peppy here.
+
+osu! to me has always been about freedom of speech and action. I've always strived to continue running the service with the minimum possible level of moderation, intervention and imposition on the players and the team.
+
+In the last week, we have had some players come forward and describe events that happened between themselves and prominent community members (often in voluntary team positions like the Global Moderators or Beatmap Approval Team) which can objectively defined as "wrong" from a moral standpoint, involving inappropriate exchanges where one party is under the age of 18.
+
+An important detail is that while multiple reports arrived in the last week, the events described occurred years ago. This shows that the users either did not feel it was necessary to report, or did not feel comfortable with reporting at the time it occurred.
```

Comment:
this segment reads a bit like you're blaming the victims for "did not feel it was necessary to report"

often times abuse is reviewed in hindsight or when experiences are shared between people, many often think what they're going through is perfectly normal at the time and don't discover the issue until later on.

would just omit this whole sentence or reshape it entirely to state that despite the reports detailing acts that happened years ago, we still acted on them because it was the right thing to do, and that others should seek to do the same if they find themselves in similar situations

Reply:
this is literally here to explain why we are changing things. it needs to be explained to give context. if you feel like this is somehow placing blame, please reword in a way you believe doesn't.

Reply:
```suggestion
An important detail is that while multiple reports arrived in the last week, the events described occurred years ago. This highlights the grave inadequacies in our internal policies and failings of our culture to catch this kind of behavior, something that we are now striving to address while still keeping our core values in tact.
```

something like this?

Reply:
how's this:

```suggestion
An important detail is that while multiple reports arrived in the last week, some events described occurred up to 9 years ago. The fact we are only learning about this now may be due to our lack of policy on when to report, how to handle and the limits of what is acceptable behaviour in the osu! community.
```

Reply:
much better!